flocker/provision/_aws.py

```diff
@@ -473,7 +413,7 @@ def make_node(ignored, name, instance):
             # so we can leave action_completion alone now.
             return results
 
-    def _async_get_node(self, reactor, instance, metadata):
+    def _get_node(self, reactor, instance, metadata):
         """
         Configure the given AWS instance, wait until it's running
         and create an ``AWSNode`` object for it.
```

Comment:
This isn't an issue with your change but I find this docstring a little confusing since an `AWSNode` object isn't created as part of this function. That appears to happen within `make_node` in `create_nodes`.